backend/internal/repository/mongo_dto/task_list.go

```diff
@@ -0,0 +1,60 @@
+package mongodto
+
+import (
+	"shodo/models"
+
+	"go.mongodb.org/mongo-driver/bson/primitive"
+)
+
+type TaskList struct {
+	ID         primitive.ObjectID `bson:"_id,omitempty"`
+	Title      string             `bson:"title,omitempty"`
+	Owner      primitive.ObjectID `bson:"owner,omitempty"`
+	SharedWith []primitive.ObjectID
+	Tasks      []Task `bson:"tasks,omitempty"`
+}
+
+func (this *TaskList) New() {
+	this.ID = primitive.ObjectID{}
+	this.Title = ""
+	this.Owner = primitive.ObjectID{}
```

Comment:
New like this is basically a reset. So why not call it like that?
New signals creation of a type, and you are not creating a type here :relaxed: